modules/web/server/src/main/resources/app.conf

```diff
@@ -40,7 +40,7 @@ web-server {
 # Configuration of the observe engine
 observe-engine {
     # host for the odb
-    odb = localhost
+    odb = "wss://odb.gpp.lucuma.xyz/ws"
```

Comment:
This seems to be the staging instance of the ODB. I'm not sure which one we want to point from here, but there's also a development instance in `https://lucuma-odb-development.herokuapp.com/`.